hutch_python/tests/test_cli.py

```diff
@@ -24,6 +25,7 @@ def no_op(*args, **kwargs):
     monkeypatch.setattr(hutch_python.cli, 'start_ipython', no_op)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Does not run on Windows")
```

Comment:
Just a suggestion, but maybe:
conftest.py -> `skip_if_win32 = pytest.mark.skipif(sys.platform == "win32", reason="Does not run on Windows")`

Then `@skip_if_win32`

Reply:
That's a good idea